.requirements

```diff
@@ -9,8 +9,9 @@ LIBEXPAT=2.5.0
 LUA_KONG_NGINX_MODULE=691ba795ced07364d491e8abbdf0c8c8d3778c14 # 0.10.0
 LUA_RESTY_LMDB=7d2581cbe30cde18a8482d820c227ca0845c0ded # 1.4.2
 LUA_RESTY_EVENTS=8448a92cec36ac04ea522e78f6496ba03c9b1fd8 # 0.2.0
-LUA_RESTY_WEBSOCKET=60eafc3d7153bceb16e6327074e0afc3d94b1316 # 0.4.0
+LUA_RESTY_WEBSOCKET=c9e2a36dca0c8c12a306071bc8223c868f530f97 # feat/get_resp_headers
```

Comment:
What should we do before we merge this PR into master? Should we create a patch?

Reply:
Certainly 👍 .

Reply:
I created a new PR for it, see: https://github.com/Kong/lua-resty-websocket/pull/16